vendor/github.com/docker/docker/client/image_create.go

```diff
@@ -21,6 +22,9 @@ func (cli *Client) ImageCreate(ctx context.Context, parentReference string, opti
 	query := url.Values{}
 	query.Set("fromImage", reference.FamiliarName(ref))
 	query.Set("tag", getAPITagFromNamedRef(ref))
+	if options.Platform != "" {
+		query.Set("platform", strings.ToLower(options.Platform))
+	}
```

Comment:
is this change related to projectatomic->containers rename?

Reply:
same comment for some other diffs below.

Reply:
These were vendored in to make the build work.